cpu/stm32l1/ldscripts/stm32l151rc.ld

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ * Copyright (C) 2015 Hamburg University of Applied Sciences
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_stm32l1
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for the STM32L151RC
+ *
+ * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author          Katja Kirstein <katja.kirstein@haw-hamburg.de>
```

Comment:
Some days ago I discussed with Hauke, that it's fine to replace the author if a file is just used as a "skeleton" or "template". Here I think you can apply these outcomes :-) . (Especially as the code is just 2 lines long)